lstchain/calib/camera/drs4.py

```diff
@@ -13,23 +16,32 @@
 roi_size = 40
 high_gain = 0
 low_gain = 1
+N_module_in_camera = 265
```

Comment:
right now this number is hardcoded, if we have observations with less modules how can you change it?
For the naming:
N_module_in_camera -> n_modules_in_camera

Reply:
This variable is used to create array with pedestal values. (I need it to avoid run out of range by numba function, when module is missing) To iterate over module code uses `self.n_module` read from data file.
Naming: I have changed.